renku_sphinx_theme/static/custom.css

```
@@ -1,63 +1,78 @@
-/*!
- * Copyright 2017 - Swiss Data Science Center (SDSC)
- * A partnership between École Polytechnique Fédérale de Lausanne (EPFL) and
- * Eidgenössische Technische Hochschule Zürich (ETHZ).
- *
- * Licensed under the Apache License, Version 2.0 (the "License");
- * you may not use this file except in compliance with the License.
- * You may obtain a copy of the License at
- *
- *     http://www.apache.org/licenses/LICENSE-2.0
- *
- * Unless required by applicable law or agreed to in writing, software
- * distributed under the License is distributed on an "AS IS" BASIS,
- * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
- * See the License for the specific language governing permissions and
- * limitations under the License.
- */
-
- 
+/**
+
+
+
+  CHANGE STYLE RELATED THINGS HERE!!!
+
+  
+
+  Requirements: 
+
+  - pip install compile-scss https://pypi.org/project/compile-scss/
+
+
+
+  Steps:
+
+  - Run:  compile_scss --watch
+
+  - That command will compile scss changes into css.
+
+  - After doing changes commit scss and css files.
+
```

Should this file not be changed, since it is generated from the sass?
```suggestion
  DO NOT CHANGE THIS FILE!!! IT IS GENERATED FROM *custom.scss*
  
```

---

yes, i realised that the compiler puts the comments as well... i will change the comment to 
" CHANGES FOR CUSTOM.CSS SHOULD BE MADE INSIDE CUSTOM.SCSS"